src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  box-sizing: border-box;
```

Comment:
Better practice is to set this property to the * selector to avoid code duplication.

Reply:
Remove from all other places.

Reply:
move to index.scss